lib/govuk_app_config/govuk_error/configuration.rb

```diff
@@ -12,32 +12,32 @@ def initialize(_raven_configuration)
       @data_sync = GovukDataSync.new(ENV["GOVUK_DATA_SYNC_PERIOD"])
       self.active_sentry_environments = []
       self.data_sync_excluded_exceptions = []
-      self.should_capture = ignore_exceptions_based_on_env_and_data_sync
+      @before_send_callbacks = [
+        ignore_exceptions_based_on_env_and_data_sync,
```

Comment:
Now that this is an array, I don't think there's value in having a separate `ignore_exceptions_based_on_env_and_data_sync` method. I think we can just pass `ignore_exceptions_if_not_in_active_sentry_env` and `ignore_excluded_exceptions_in_data_sync` in here directly.